### ISSUE TITLE
Silent Error During Build

### ISSUE THREAD

**Issue by makeclean** [pyne/pyne#915]
Silent error during building introduced recently, 

```Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: getsitepackages() takes no arguments (1 given)
```
Can be seen in CI since swtich to Circle. No idea if it has any side effects.

**Comment by scopatz** [comment]
Thanks for reporting @makeclean - that error shouldn't be silent.  Can you provide a link to failing case on Circle? I bet this is a Python 2 issue.

**Comment by ljacobson64** [comment]
This was fixed by #931
